src/sentry/search/events/builder/metrics.py

```diff
@@ -1444,51 +1444,41 @@ def run_query(self, referrer: str, use_cache: bool = False) -> Any:
             )
 
             try:
+                metrics_queries = []
```

Comment:
This `run_query` branch is mostly the same as what we did for top events, except the queries being appended to the queries array is obviously different.